qtpy/_patch/qheaderview.py

```diff
@@ -0,0 +1,48 @@
+def introduce_renamed_methods_qheaderview(QHeaderView):
+
+    def sectionsClickable(self):
+        """
+        QHeaderView.sectionsClickable() -> bool
+        """
+        return QHeaderView.isClickable(self)
+    QHeaderView.sectionsClickable = sectionsClickable
```

Comment:
Why define a new method each time ? This code should be equivalent:
```python
QHeaderView.sectionsClickable = QHeaderView.isClickable
```
I think there a re some differences, like the `__name__` attribute would still be the one from the initial method. On the other hand the help should be more complete (instead of just the signature in the current state of this PR).